lib/prompts/summarizeText.ts

```diff
@@ -0,0 +1,41 @@
+import { ChatGPTMessage } from "../models";
+import { Organization } from "../types";
+
+export const summariseEmailGPTParams = {
+  temperature: 0.7,
+  max_tokens: 400,
+  frequency_penalty: 0.5,
+  presence_penalty: 0,
+};
+
+export function getSummarizeTextPrompt(
+  textToSummarize: string,
+  organization: Organization
+): ChatGPTMessage[] {
+  return [
+    {
+      role: "system",
+      content: `Your task is to summarize a document using bullet points and short simple sentences.
+
+For context of what to include in your summary, you are working for ${organization.name}. ${organization.description}
+
+Your response should be a maximum of 6 bullet points. Only use one, two or three bullet points if you can. Only write short simple sentences. Only write one sentence per bullet point. Be as succinct as possible.
```

Comment:
This is probably hard but the summary often doesn't contain the information that you'd actually want to see:

**original**:
![image](https://github.com/Superflows-AI/superflows/assets/28537547/03c8f0d5-d3b3-4a40-972a-cc89c598b4dd)

**Summary**
![image](https://github.com/Superflows-AI/superflows/assets/28537547/f6c64fa6-a5c1-4f36-9292-9e38fde77d72)

I would want the summary to say e.g. 
"the net assests for the company is X" rather than "Balance sheet shows fixed assets, current assets, and net assets."

Reply:
maybe we need a line like: "be specific about the facts you include, include numerical information"

Reply:
Surprised it came out with this! Never seen it do this

Have this line already:
https://github.com/Superflows-AI/superflows/blob/summarise-long-function-responses/lib/prompts/summarizeText.ts#L24

Added a `THIS IS VERY IMPORTANT. DO NOT FORGET THIS!`

Reply:
oh yeah that worked nicely when i just tried it